core/src/main/scala/akka/persistence/cassandra/query/EventsByPersistenceIdStage.scala

```diff
@@ -329,6 +334,14 @@ import com.datastax.driver.core.utils.Bytes
                 if (pendingPoll.get >= seqNr)
```

Comment:
I think the above `lookingForMissingSeqNr.isEmpty` is correct but perhaps we should add another gap test for the `currentEventsByPersistenceId` query, i.e. when `refreshInterval.isEmpty` to be sure that the stage is completed in that case

Reply:
Is the correct behavior to complete the stream then, rather than failing it with the "saw gap" exception?

Reply:
Good question, if the reason is a delay then it would be just as fine to complete the stream, but if the reason is that the event is actually missing it would be better to fail the stream if it doesn't show up within the timeout.

Reply:
Hmmm. I think it is better to have it fail then, so at least it is noticed that something is potentially wrong, retrying a bit later would then not fail if the gap was because of a delayed event.

Reply:
Sounds good. Do you mean fail immediately when noticing a gap or try to looking for the missing in the same way as the live query. I think the latter could be better, especially if it just works like that in current implementation.

Reply:
The latter was how it already works, added another test case for `current` when the timeout doesn't hit and the missing event come in and that fails, so need to figure that one out.

Reply:
thx